02/app.js

```diff
@@ -1,7 +1,28 @@
 
 /* rozwiązanie z pętlą for */
-const x = 4;
+const x = prompt('Podaj liczbę: ');
+
+if(x >= 1 && x <= 9){
+for (let i = 0; i <= 9; i++) {
+    console.log(`${x}*${i} to ${x*i}`);
+}
+} else {
+    alert('Błąd!');
+}
+
+
+
+/* rozwiązanie z pętlą while  */
+
+let a = prompt('Podaj podstawę: ');
+let b = prompt('Podaj wykładnik: ');
```

Comment:
Podobnie tutaj. Zawsze dane od użytkownika - czy to przez formularz lub w taki sposób są typu `string`